convert/rulesets/280-to-340/plugins.yaml

```diff
@@ -0,0 +1,98 @@
+rules:
```

Comment:
@Prashansa-K what would our source of truth be to come up with these rules - ie, how do we know what has changed for each entity from gateway version `a` to `b`?
I can think of `/schemas/{entity}` endpoint on admin API, is there anything else?

Reply:
https://github.com/Kong/kong-lts-migration-preflight/blob/main/deck-rulesets/28-to-34/rules/plugins.yaml
We are using the same rules that the TSE teams use.